article.md

```diff
@@ -87,11 +94,18 @@ async def main(title: str, location: str, data_name: str) -> None:
         "position": "1",
         "pageNum": "0"
     }
-    encoded_params = urllib.parse.urlencode(params)
-    encoded_url = f"{base_url}?{encoded_params}"
+
+    encoded_params = urlencode(params)
+    
+    # Encode parameters into a query string
+    query_string = '?' + encoded_params
+
+    # Combine base URL with the encoded query string
+    encoded_url = urljoin(base_url, "") + query_string
```

Comment:
IMO this is not better than the previous solution. If there's no nice way to do what I wanted in Python, you could've just said so ;) Let's use the original version.